plugins/bigquery/dbt/include/bigquery/macros/adapters.sql

```diff
@@ -36,24 +36,25 @@
   {% if temporary %}
     {% do opts.update({'expiration_timestamp': 'TIMESTAMP_ADD(CURRENT_TIMESTAMP(), INTERVAL 12 hour)'}) %}
   {% endif %}
+  {% if kms_key_name %}
+    {% do opts.update({'kms_key_name': "'" ~ kms_key_name ~ "'"}) %}
```

Comment:
there is _a lot_ we can do to make these settings less onerous to manage, especially if we intend to add more of them in the future. For now though, this is a-ok with me